Source/santactl/Commands/SNTCommandMonitorMode.mm

```diff
@@ -53,8 +53,67 @@ + (NSString *)longHelpText {
   return [NSSet setWithArray:@[ @"mm" ]];
 }
 
+// Parse a time interval string into a number of minutes.
+// e.g. "10m" -> 10, "2h" -> 120, "3d" -> 3600
+- (NSTimeInterval)parseTimeInterval:(NSString *)duration {
+  NSScanner *scanner = [NSScanner scannerWithString:duration];
+  scanner.charactersToBeSkipped = nil;
+  NSString *unit = nil;
+
+  NSInteger intValue = 0;
+  NSUInteger locationBeforeScan = scanner.scanLocation;
+  if ([scanner scanInteger:&intValue]) {
+    NSUInteger locationAfterInteger = scanner.scanLocation;
```

Comment:
This is overkill... I don't think it's possible for the scan location to not progress if success is returned, rendering much of the new code here inaccessible. Think the previous commit is fine: https://github.com/northpolesec/santa/pull/675/commits/7ea8cfe2da186293f298c8adcd3b375f68353fd2